<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Member</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight:bold;
font-size:14px;</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioTeacher">
          <property name="text">
           <string>Teacher</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioNonTeacher">
          <property name="text">
           <string>Non-teaching staff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioPlusOne">
          <property name="text">
           <string>Plus one</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioPlusTwo">
          <property name="text">
           <string>Plus two</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="name">
      <property name="styleSheet">
       <string notr="true">color:#000000;
background:#ffffff;
border:1px solid #55000000;
border-radius:10px;
padding:5px;</string>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="id">
      <property name="styleSheet">
       <string notr="true">color:#000000;
background:#ffffff;
border:1px solid #55000000;
border-radius:10px;
padding:5px;</string>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(0, 170, 255);
	border-radius:10px;
	padding:5px;
	color:#ffffff;
}
QPushButton:hover{
	background:rgb(50, 120, 205)
}</string>
      </property>
      <property name="text">
       <string>ADD MEMBER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
